eng/mgmt/mgmtmetadata/attestation_data-plane.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
```

Comment:
Having this file under `mgmt/mgmtmetada/` doesn't seem right to me. @AlexGhiondea  Hi Alex, do you happen to know the excepted behavior for data plane?

Reply:
@isra-fel according to the docs, this file is expected.  [Link](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)

>generate.ps1 will create a metadata file under eng\mgmt\mgmtmetadata\<rpName>.txt, have that file as part of your PR.

Reply:
As this is an auto-generated project, I believe the file is expected.